test/conftest.py

```diff
@@ -0,0 +1,11 @@
+import pytest
+import numpy as np
+from kmapper import KeplerMapper
+
+@pytest.fixture
+def sc():
```

Comment:
what does `sc` stand for?

Reply:
predates me, but simplicial complex, I assume. I moved this from a different test file into conftest.py so that all tests could use it.